src/shared/components/case-list-filters/case-list-filters.component.html

```diff
@@ -0,0 +1,41 @@
+<ng-container *ngIf="jurisdictions">
```

Comment:
This is supposed to wrap the workbasket filters component similar to search filter having a wrapper component. Not sure why we have a copy here?

Reply:
Done.